---
layout: default
title: Glossary
nav_order: 9
has_children: false
---

# Glossary
{: .no_toc }

<details open markdown="block">
  <summary>
    Table of contents
  </summary>
  {: .text-delta }
1. TOC
{:toc}
</details>

## Population size

### Census Size, Nc
The number of reproductively mature individuals in a population. This number can be estimated via physical counts- Itt should be noted that these counts are not necessarily exact numbers of individuals, but rather estimates given that the detection probability is dependent on several factors, including population density, observer effort (time), and experience. There are instances where it is impractical or impossible to estimate the size of a population by traditional counting methods. In such instances, additional  methods, such as capture-mark-recapture, can be used. An estimate based on density in a given unit area and the total area can be used, if it can be assumed that the area is occupied. Estimates can also be made by local knowledge holders based on personal experience.

### Effective population size, Ne
The effective population size is the genetic complement of **census size**; where census size influences ecological aspects of a population, genetic factors of a population are influenced by Ne. Both random [allele](https://ccgenetics.github.io/guidelines-genetic-diversity-indicators/docs/7_Glossary/Glossary.html#allele) frequency change and estimates of inbreeding are influenced by Ne. For the purposes of the genetic diversity indicators, Ne refers to contemporary effective population size which measures ongoing and near-term genetic drift. Estimating contemporary Ne does not require knowing mutation rates (which is required for estimating historic Ne), but does require genetic data OR a demographic knowledge (e.g. **census size, Nc**), and a “rule of thumb” to convert Nc to Ne, i.e. **Ne/Nc ratio**, which in many cases  is close to 0.1 (Frankham 1995; and see Hoban et al. 2020, 2021).

### Ne/Nc ratio
Relationship between the **census size (Nc)** and the **effective population size (Ne)**. This ratio enables the estimation of effective population size when only census size is known, or vice versa. It is a helpful metric for calculating genetic diversity indicators, where census size may be the most common estimate of population size. Where available, species specific ratios enable the best estimates of Ne from Nc. Where unavailable, ‘rule of thumb’ values can be used to estimate Ne (Frankham 1995; and see Hoban et al. 2020, 2021)

### 50/500 rule 
The 50/500 rule has been used as a guiding principle in conservation for assessing minimum viable **effective population size (Ne)** (Soule 1980; Franklin 1980). The overall premise behind this rule is that in order for a species or population to persist, a minimum Ne of 50 is needed in the short term to combat inbreeding, while a minimum Ne of 500 in the long term is needed to reduce genetic drift and enhance/promote evolutionary potential in perpetuity.  A recommendation to double these minimum Ne estimates was made, increasing Ne to 100 and 1000 respectively (Frankham et al. 2014). While some have adopted this change, the 50/500 rule remains more widely used. 


## Genetics

### Allele
A variant in the DNA sequence at a given gene. For example CCG and CCT are alleles.  A given gene may have a few or many alleles.  Some alleles alter the phenotype and fitness (adaptive) and some do not (neutral).  Alleles are also sometimes called “mutations.”

### Genetic drift 
Genetic drift is the random  change in the genetic composition of a population over time (generations). For populations not connected by **gene flow**, genetic drift can result in differences in genetic composition between populations that are the result of chance, rather than adaptation to local environment(neutral evolutionary processes).

### Gene flow 
The exchange of genetic material between two populations, through migration of gametes (e.g. pollen) or individuals (e.g. offspring) from one population into another. For gene flow to occur, individuals entering the population must contribute genes to the population, i.e. reproduce and create offspring within that population. Gene flow has a homogenizing effect on genetic composition, countering the effects of **genetic drift**. Populations connected by gene flow have more similar genetic composition than populations with limited or no gene flow.

### Isolation by distance
Isolation by distance describes a pattern of genetic variation across a species’ range where the genetic similarity between populations decreases with increasing geographic distance. This is due to the effects of **genetic drift** and **gene flow**. As geographic distance increases between populations, gene flow decreases, increasing genetic drift and the dissimilarity in genetic composition between populations. 

### Genetic markers
The form of genetic variation used to measure DNA-level diversity within the population or species. Multiple genetic markers exist for assessing genetic diversity at the DNA-level including single nucleotide polymorphisms (SNPs), microsatellites, and chloroplast or mitochondrial DNA sequences. All markers are valid and useful for estimating genetic diversity and Ne, with some differences in the type and level of information each can provide.

### Hybridization
The interbreeding between individuals from two genetically distinct populations, lineages, or species, producing offspring with mixed ancestry.

### Introgression
The incorporation of genetic material from one population, lineage, or species into another through repeated backcrossing of hybrids with one of the parental groups.

### Genetic studies 

- #### Phylogenetic studies
use genetic data to understand the relatedness among groups of organisms, e.g. among species or among subspecies within species. They aim to understand the evolutionary history and relationship between entities within the group, determining entities that are more closely or more distantly related. They typically use only a few genetic samples for a given focal species.

- #### Phylogeographic studies
investigate the geographic distribution of the different entities (evolutionary lineages). In addition to relatedness among entities, it explores how historic processes, geographic and genetic, have led to the current distribution of lineages across a landscape. They typically use several genetic samples (individuals) of several populations within the species range.

- #### Population-level studies
focus on understanding variation within and between populations of a species. They use genetic information to assess a range of population metrics including, but not limited to, genetic diversity within populations, genetic divergence (dissimilarity) among populations, levels of inbreeding, and effective population size (Ne). 

- #### Temporal genetic monitoring
uses sampling and analysis of genetic variation at the population level across at least two time points, typically separated by one or more generations or cohorts (i.e., reflecting genetic change over time).

## Species distributions

### National endemic
A species whose whole distribution occurs within a single country, i.e. is only found in a single country. For national reporting of genetic diversity indicators, a country would be reporting on the entire species distribution.

### Transboundary distribution
A species whose distribution spans multiple countries. For national reporting of genetic diversity indicators, an individual country would therefore be reporting on only a portion of the full species distribution.

### Extent of Occurrence
Extent of occurrence (EOO), as defined by the IUCN, is the general region a species occurs in, including all known, inferred or projected sites. It is calculated as the area of a convex hull around the whole distribution- basically a polygon around the whole range. EOO reflects whether a species is found across a large or small distribution. 

### Area of Occurrence
Area of Occurence (AOO) is the area within a species’ **Extent of Occurrence** which is actually occupied by the species. It is calculated as the area of the sum of grid cells a species occurs in. AOO reflects how often you find the species within its large distribution. A species can have a large EOO but a small AOO.

### Species range 

- #### Wide-ranging (or widespread) species
have an extent of occurrence (EOO) greater than 20 000 〖km〗^2 or are present in more than one-third of a country's 20-km grid squares. 

- #### Restricted range species
are those that have an EOO <20 000 km2 or an area of occurrence (AOO) < 2 000 km2. A list of restricted species has been prepared using KBA criteria and can be found [here](https://www.keybiodiversityareas.org/working-with-kbas/proposing-updating/criteria-tools); however, this list should only be used as a guide as not all species have been assessed. 

- #### Rare species
are those that occur in low densities, typically as single individuals or as very small subpopulations with fewer than 50 individuals. They may occur within a restricted range, but are typically scattered over a wide area.

### Realm 
High-level category of the environment a species inhabits. Four geographic realms are recognized - marine, estuarine, freshwater, terrestrial - though some species may inhabit multiple realms. For example, many amphibians exist in the terrestrial and freshwater realms.

### IUCN Habitat Classification
A list of general, major habitats that species occur in, globally. Within each **realm**, species may occur in one or more major habitat types. For a detailed explanation on each habitat type and the sub-types within them, please refer to the [IUCN Habitat Classification Scheme](https://www.iucnredlist.org/resources/habitat-classification-scheme). 


## Species' biology 

### Population
In general, a population is a group of individual organisms that can mate with each other and have low or no connectivity with other populations. However, this definition can vary depending on the species. See [‘What is a population’](https://ccgenetics.github.io/guidelines-genetic-diversity-indicators/docs/2_Theoretical_background/What-is-a-population.html#what-is-a-population-a-first-simple-answer) under [‘Theoretical background’](https://ccgenetics.github.io/guidelines-genetic-diversity-indicators/docs/2_Theoretical_background/Theoretical-background.html#background) for detailed discussion of what a population is and how to define a population for assessments of genetic diversity indicators.

### Metapopulations
A metapopulation consists of numerous groups of individuals (subpopulations), occupying multiple localities (e.g. ponds, prairies, forests etc.), that are separate but within the species’ dispersal capacity. Subpopulations within a metapopulation are thus capable of exchanging at least one migrant (one reproductive adult moving between patches) on average per generation per year. Metapopulations should represent stable spatial and temporal units. In the case of a metapopulation, the population size should be considered the sum of the individual subpopulations.

### Closely related species
For the purpose of calculating genetic indicators, closely related species are those species who share a more recent common ancestor, compared to other species, e.g. species in the same genus or sub-genus. Such species will be more genetically similar to each other than other, less related species and may share some life history similarities.

### Origin of population 

- #### Historically natural populations
Populations where the native gene pool has been in that location for ‘some generations’ (e.g. before human influences). The genetic variation within these populations has developed through natural evolutionary processes and represents a unique, native gene pool within the species.

- #### Introduced populations
Populations that in the recent past (e.g. past hundreds of years) have never been known to occur at that particular location. Now they exist there, due to active translocation of individuals. Depending on the source of the introduced individuals, the genetic composition of the introduced population may not represent a unique genetic population from a genetic diversity indicator perspective. Rather, the genetic composition should be considered an expansion of the source population(s), as its genetic composition reflects the source population(s), not the naturally occurring genetic composition at the location.

- #### Re-introduced  populations
Populations that went extinct in a location and have since been reintroduced, by humans or, more rarely, through natural processes. Depending on the source of the re-introduced individuals, the genetic composition of re-introduced populations may differ from the original genetic composition at that location. Reintroductions can be done from captive breeding with small population sizes, thus likely presenting a mixed population origin, or sourcing individuals from another location. This will influence whether the original population, with its unique genetic composition, may be considered extant (noting potential genetic changes due to bottlenecks), or not.

- #### Supplemented (aka reinforced)
Populations are naturally occurring populations that have a native gene pool but have had new individuals introduced to support the population. Depending on the origin of the new individuals, e.g. from a breeding population or other population, the reinforced population may contain non-local genetic material. The genetic composition of such populations thus can be a  combination of the native gene pool and non-native variation from other populations.

- #### *Ex situ* collections
Collections of living organisms (or their seeds or tissue) that are maintained outside the natural habitats of those organisms for the purpose of ensuring their survival and future propagation. Because the genetic diversity indicators are meant to provide an assessment of the genetic status of wild populations, *ex situ* collections (and the individuals within them) should generally not be included in indicator calculations. 
 
[Previous: Calculations and reporting](https://ccgenetics.github.io/guidelines-genetic-diversity-indicators/docs/6_Calculations_and_reporting/Calculations_and_reporting.html#calculations-and-reporting){: .btn .btn-blue .mr-4 }
[Next: Reference and other resources](https://ccgenetics.github.io/guidelines-genetic-diversity-indicators/docs/Video_resources/References_and-resources.html#references-and-other-resources){: .btn .btn-green }
